doc/source/data/pipelining-compute.rst

```diff
@@ -0,0 +1,120 @@
+.. _pipelining_datasets:
+
+==================
+Pipelining Compute 
```

Comment:
In a future PR, we might want to look at consolidating this page and the "Advanced Pipeline Usage" page; the latter contains some fundamental things about pipelining that should probably be in this page.